x/evm/atlas/atlas-v0.3.1.md

```diff
@@ -0,0 +1,181 @@
+# x/evm
+
+The `x/evm` module is responsible for executing Ethereum Virtual Machine (EVM) state transitions.
+
+## Usage
+
+1. Import the module and the dependency packages.
+
+   ```go
+   import (
+      "github.com/cosmos/cosmos-sdk/x/auth"
+      "github.com/cosmos/cosmos-sdk/x/bank"
+       
+      "github.com/cosmos/ethermint/app/ante"
+      ethermint "github.com/cosmos/ethermint/types"
+      "github.com/cosmos/ethermint/x/evm"
+   )
+   ```
+
+2. Add `AppModuleBasic` to your `ModuleBasics`.
+
+    ```go
+    var (
+      ModuleBasics = module.NewBasicManager(
+        // ...
+        evm.AppModuleBasic{},
+      }
+    )
+    ```
+
+3. Create the module's parameter subspace in your application constructor.
+
+   ```go
+   func NewApp(...) *App {
+     // ...
+     app.subspaces[evm.ModuleName] = app.ParamsKeeper.Subspace(evm.DefaultParamspace)
+   }
+   ```
+
+4. Define the Ethermint `ProtoAccount` for the `AccountKeeper`
+
+   ```go
+   func NewApp(...) *App {
+      // ...
+        app.AccountKeeper = auth.NewAccountKeeper(
+            cdc, keys[auth.StoreKey], app.subspaces[auth.ModuleName], ethermint.ProtoAccount,
+        )
+   }
+   ```
+
+5. Create the keeper.
+
+   ```go
+   func NewApp(...) *App {
+      // ...
+        app.EvmKeeper = evm.NewKeeper(
+         app.cdc, keys[evm.StoreKey], app.subspaces[evm.ModuleName], app.AccountKeeper,
+        )
+   }
+   ```
+
+6. Add the `x/evm` module to the app's `ModuleManager`.
+
+   ```go
+   func NewApp(...) *App {
+     // ...
+     app.mm = module.NewManager(
+       // ...
+       evm.NewAppModule(app.EvmKeeper, app.AccountKeeper),
+       // ...
+     )
+   }
+   ```
+
+7. Set the `x/evm` module `BeginBlock` and `EndBlock` ordering:
+
+    ```go
+      app.mm.SetOrderBeginBlockers(
+        evm.ModuleName, ...
+      )
+      app.mm.SetOrderEndBlockers(
+        evm.ModuleName, ...
+      )
+    ```
+
+8. Set the `x/evm` module genesis order. The module must go after the `auth` and `bank` modules.
```

Comment:
is there supposed to be explicit instructions on how to set up the `auth` and `bank` modules?